### PR TITLE
aggreate raw instance metrics

### PR DESCRIPTION
**Pull request by boyang9527**
[#131839361]

- return raw instance metrics when aggregator queries metrics from metrics collector
- use raw instance metrics to aggregate
- responding changes when change from metric to appinstancemetric

### CODE REVIEW

File: src/autoscaler/eventgenerator/aggregator/metricPoller.go [hunk 1]
```diff
@@ -86,18 +86,16 @@ func (m *MetricPoller) retrieveMetric(app *model.AppMonitor) {
...
 	var timestamp int64 = time.Now().UnixNano()
 	for _, metric := range metrics {
 		unit = metric.Unit
-		for _, instanceMetric := range metric.Instances {
-			count++
-			intValue, _ := strconv.ParseInt(instanceMetric.Value, 10, 64)
-			sum += intValue
-		}
+		count++
+		intValue, _ := strconv.ParseInt(metric.Value, 10, 64)
```
Comment (fraenkel) [inline src/autoscaler/eventgenerator/aggregator/metricPoller.go]: Shouldn't we log when we get an error?

File: src/autoscaler/metricscollector/collector/apppoller.go [hunk 2]
```diff
@@ -72,21 +72,19 @@ func (ap *appPoller) startPollMetrics() {
 func (ap *appPoller) pollMetric() {
 	ap.logger.Debug("poll-metric", lager.Data{"appid": ap.appId})
 
-	containerMetrics, err := ap.noaaConsumer.ContainerEnvelopes(ap.appId, "bearer"+" "+ap.cfc.GetTokens().AccessToken)
+	ContainerEnvelopes, err := ap.noaaConsumer.ContainerEnvelopes(ap.appId, "bearer"+" "+ap.cfc.GetTokens().AccessToken)
```
Comment (fraenkel) [inline src/autoscaler/metricscollector/collector/apppoller.go]: why is it capitalized?

Comment (boyang9527) [inline src/autoscaler/metricscollector/collector/apppoller.go]: it is a typo

**Comment by boyang9527** [comment]
@fraenkel  I pushed another commit to fix the issues.  please review.

**Pull request merged**